src/helm/benchmark/scenarios/mmlu_pro.py

```diff
@@ -0,0 +1,93 @@
+import csv
+import os
+from typing import Dict, List
+
+from helm.common.general import ensure_file_downloaded, ensure_directory_exists
+from helm.common.hierarchical_logger import hlog
+from .scenario import Scenario, Instance, Reference, TRAIN_SPLIT, VALID_SPLIT, TEST_SPLIT, CORRECT_TAG, Input, Output
+
+import datasets
+from typing import List, Dict
+from pathlib import Path
+import json
+import random 
+
+
+from helm.benchmark.scenarios.scenario import (
+    Scenario,
+    Instance,
+    Reference,
+    TEST_SPLIT,
+    Input,
+    Output,
+)
+from helm.common.general import ensure_directory_exists
+
+class MMLUPROScenario(Scenario):
+    """
+    This is sample code for the MMLU-PRO Dataset.
```

Comment:
Fill in a description of the dataset. See the other scenarios for a general idea of what to do. You can use a summary of the abstract, and you should cite the paper and link to the paper and dataset.